packages/camera/ios/Classes/CameraPlugin.m

```diff
@@ -246,6 +235,59 @@ - (void)captureToFile:(NSString *)path result:(FlutterResult)result {
                                                            cameraPosition:_captureDevice.position]];
 }
 
+- (void)setCaptureSessionPreset:(NSString *)resolutionPreset {
```

Comment:
This will most likely just be a temporary fix until we expose available resolution for each device.